main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -338,6 +338,11 @@ class Flix {
     */
   var options: Options = Options.Default
 
+  /**
+   *  The current Flix safety signature.
+   */
+  var safe: Boolean = true
```

Comment:
I don't understand why the Boolean is here. I though this property was per package.

Reply:
This will be corrected in the new PR when we carry the manifest with the input.